GameData/ROCapsules/PartConfigs/Vostok/VoskhodCapsule.cfg

```diff
@@ -1,195 +1,205 @@
+	//	============================================================================
+
+	TechRequired = unlockParts
+	cost = 150
+	entryCost = 1
+	category = Pods
+	subcategory = 0
 	title = Voskhod Descent Module
 	manufacturer = OKB-1
 	description = The Voskhod SA Descent Module was a re-purposed version of the Vostok SA. It was retrofitted to allow for an Airlock attachment for EVA and even to fit 3 cosmonauts. However, with 3 cosmonauts, there was not enough room to wear pressure suits. The SA stands for Spuskaemiy Apparat (Descent System) and was designed as a ballistic reentry capsule. [Diameter 2.3 m]
-	
-	mass = 2.325
-	
-	category = Pods
-	subcategory = 0
-	
 	tags = soviet, capsule, crew, descent, voskhod
-	
-	TechRequired = unlockParts
-	cost = 150
-	entryCost = 1
-	
+
 	vesselType = Ship
 	stagingIcon = COMMAND_POD
 	CrewCapacity = 3
-	
-	INTERNAL
-	{
-		name = SoyuzPod
-	}
-	
-	//  ============================================================================
+	bulkheadProfiles = size1
+
+	//	============================================================================
 	//	Thermo, Durability
-	//  ============================================================================
-		
+	//	============================================================================
+
+	mass = 2.325
+	dragModelType = default
+	maximum_drag = 0.2
+	minimum_drag = 0.2
+	angularDrag = 2
 	crashTolerance = 14
-	maxTemp = 2400
-	skinMaxTemp = 3600
-	emissiveConstant = 0.4 // not too absorptive for reentry
+	breakingForce = 250
+	breakingTorque = 250
+	fuelCrossFeed = True
+
+	heatShieldDiameter	= 2.3
+	heatShieldTag = LEO
+	resetHeatShieldAblator = true
+	resetHeatShieldMass = false
+	emissiveConstant = 0.6
 	thermalMassModifier = 1.0
 	skinMassPerArea = 4
-	
-	MODULE:NEEDS[!DeadlyReentry]
-	{
-		name = ModuleAblator
-		ablativeResource = Ablator
-		lossExp = -8000
-		lossConst = 0.1
-		pyrolysisLossFactor = 25000
-		ablationTempThresh = 500
-		reentryConductivity = 0.01
-	}
-	
-	MODULE:NEEDS[DeadlyReentry]
-	{
-		name = ModuleHeatShield
-		ablativeResource = Ablator
-		lossExp = -8000
-		lossConst = 0.1
-		pyrolysisLossFactor = 25000
-		ablationTempThresh = 500
-		reentryConductivity = 0.01
-		depletedMaxTemp = 1200
-	}
-	
-	RESOURCE
-	{
-		name = Ablator
-		amount = 225
-		maxAmount = 225
-	}
-	
-	//  ============================================================================
+
+	//	============================================================================
 	//	Modules and Resources
-	//  ============================================================================
-	
+	//	============================================================================
+
 	MODULE
 	{
 		name = ModuleCommand
 		minimumCrew = 0
 		RESOURCE
 		{
 			name = ElectricCharge
-			rate = 1.5 // A much higher requirement for 2-3 cosmonauts
+			rate = 1.5
 		}
 	}
-	
-	// Cannot EVA from Here - Must attach the Airlock and move into that
-	MODULE:NEEDS[RP-0]
+
+	MODULE
 	{
-		name = ModuleNoEVA
```

Comment:
Think this still needs to be here, since there is a requirement for an airlock part and that remains?

Reply:
Nope its already built in to RP-0